cob_description/urdf/sensors/sick_microscan3.gazebo.xacro

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="sick_microscan3_gazebo" params="name">
```

Comment:
this file still looks like a copy-paste of the s300 laserscanner...please adjust to the microscan specs, e.g. min/max dist, resolution and fov...